docs/documentation/logging.rst

```diff
@@ -0,0 +1,60 @@
+.. _logging:
+
+Logging
+=======
+
+Logging in audiomate is done using the standard python logging facilities.
+
+Enable Logging
+--------------
+
+By default, only messages of severity ``Warning`` or higher are printed to ``sys.stderr``.
+At some points audiomate provides more detailed info and debug information.
```

Comment:
I don't follow. You mean that audiomate is going to provide more info in the future or that there are some parts of audiomate that provide more info right now?